src/components/pickers/BasePicker.tsx

```diff
@@ -38,6 +39,8 @@ export class BasePicker<T, P extends IBasePickerProps<T>> extends BaseComponent<
 
   protected suggestionStore: SuggestionsController<T>;
   protected SuggestionOfProperType = Suggestions as new (props: ISuggestionsProps<T>) => Suggestions<T>;
+  protected loadingTimer: number;
+  protected currentPromise: PromiseLike<any>;
```

Comment:
It needs to be protected as long as the methods using it are protected. As soon as it becomes private anything that subclasses will be unable to implement their own version since the variable is private while the method is protected. Really I think most of the methods should be private and I'll need to go through and change them.